Mathlib/Analysis/Asymptotics/Asymptotics.lean

```diff
@@ -1938,12 +1938,18 @@ theorem isLittleO_const_id_cobounded (c : F'') :
   isLittleO_const_left.2 <| .inr tendsto_norm_cobounded_atTop
 #align asymptotics.is_o_const_id_comap_norm_at_top Asymptotics.isLittleO_const_id_cobounded
 
-theorem isLittleO_const_id_atTop (c : E'') : (fun _x : ℝ => c) =o[atTop] id :=
-  isLittleO_const_left.2 <| Or.inr tendsto_abs_atTop_atTop
+theorem isLittleO_const_id_cocompact [ProperSpace F'']
+    (c : E'') : (fun _x : F'' => c) =o[cocompact F''] id :=
+  isLittleO_const_left.2 <| Or.inr tendsto_norm_cocompact_atTop
+
+theorem isLittleO_const_id_atTop [LinearOrder F''] [NoMaxOrder F''] [ClosedIciTopology F'']
```

Comment:
Can you name a bunch of these results accorinding to the suggestion [here](https://leanprover.zulipchat.com/#narrow/stream/287929-mathlib4/topic/Naming.20convention/near/447491526) making things `protected` as appropriate?
```suggestion
theorem IsLittleO.const_id_atTop [LinearOrder F''] [NoMaxOrder F''] [ClosedIciTopology F'']
```